code/edgeconsent/src/main/java/com/adobe/marketing/mobile/edge/consent/Consent.java

```diff
@@ -58,24 +63,36 @@ public void error(ExtensionError extensionError) {
 	 */
 	public static void update(final Map<String, Object> consents) {
 		if (consents == null || consents.isEmpty()) {
-			MobileCore.log(LoggingMode.DEBUG, ConsentConstants.LOG_TAG,
-						   "Consent - Null/Empty consents passed to update API. Ignoring the API call.");
+			MobileCore.log(
+				LoggingMode.DEBUG,
+				ConsentConstants.LOG_TAG,
+				"Consent - Null/Empty consents passed to update API. Ignoring the API call."
+			);
 			return;
 		}
 
 		// create and dispatch an consent fragments update event
 		final ExtensionErrorCallback<ExtensionError> errorCallback = new ExtensionErrorCallback<ExtensionError>() {
 			@Override
 			public void error(final ExtensionError extensionError) {
-				MobileCore.log(LoggingMode.DEBUG, ConsentConstants.LOG_TAG,
-							   String.format("Consent - update API. Failed to dispatch %s event. Ignoring the API call. Error : %s.",
-											 ConsentConstants.EventNames.CONSENT_UPDATE_REQUEST,
-											 extensionError.getErrorName()));
+				MobileCore.log(
+					LoggingMode.DEBUG,
+					ConsentConstants.LOG_TAG,
+					String.format(
+						"Consent - update API. Failed to dispatch %s event. Ignoring the API call. Error : %s.",
+						ConsentConstants.EventNames.CONSENT_UPDATE_REQUEST,
+						extensionError.getErrorName()
+					)
+				);
 			}
 		};
-		final Event event = new Event.Builder(ConsentConstants.EventNames.CONSENT_UPDATE_REQUEST,
-											  ConsentConstants.EventType.CONSENT, ConsentConstants.EventSource.UPDATE_CONSENT).setEventData(
-			consents).build();
+		final Event event = new Event.Builder(
+			ConsentConstants.EventNames.CONSENT_UPDATE_REQUEST,
+			ConsentConstants.EventType.CONSENT,
+			ConsentConstants.EventSource.UPDATE_CONSENT
+		)
+			.setEventData(consents)
+			.build();
```

Comment:
Unfortunately no. Prettier's rules places the closing parenthesis on the next line with no indentation, while indenting the chained method calls each on their own line. This would look correct if the builder didn't have any parameters in its constructor, but with the two formatting rules together it does look a little off. Prettier has no options to configure individual rules like this so there's nothing we can do.